ToggleCipher/Package.swift

```diff
@@ -20,10 +20,34 @@ let package = Package(
             dependencies: [
                 .product(name: "ArgumentParser", package: "swift-argument-parser")
             ],
-            path: "Sources"),
+            path: "Sources",
+            swiftSettings: [
```

Comment:
@antoniostrijdom were these experimental settings intended to be added here? Only in ToggleCipher and not in ToggleGen, JustTweakMigrator and Toggles?